README.md

```diff
@@ -116,6 +116,16 @@ $ sudo apt install weston
 $ sudo apt install libdrm-dev libgbm-dev libinput-dev libudev-dev libsystemd-dev
 ```
 
+#### Only when you use EGLStream backend
```

Comment:
This is useless because it overlaps with the case of GBM.

Reply:
I deleted it.